docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
 
-It consists of four basic parts:
+It consists of several components:
 
   1. Engine - a test framework that supports all functionality to perform DNS tests.
-  2. The CLI interface to the Engine.
-  3. Backend, a server that allows you to run Zonemaster tests and save results using
+  2. CLI - a command-line interface to the Engine.
+  3. Backend - a server that allows you to run Zonemaster tests and save results using
      a JSON-RPC API and a database.
-  4. GUI - a web interface to the backend.
+  4. GUI - a web interface to the Backend.
 
-When a domain (such as "zonemaster.net") is submitted to Zonemaster (CLI or
-GUI) it will investigate the domain’s general health by traversing the DNS from root
-(.) via the TLD (Top Level Domain, like .net) to the nameserver(s) that hosts
-the information about the specific domain (zonemaster.net). The different sanity checks
-conducted by the Zonemaster tool is documented in the [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md).
+When a domain name (such as 'zonemaster.net') is submitted to Zonemaster (using CLI or
+GUI), it will verify the domain name’s general health with a series of tests.
+The different sanity checks and corresponding tests conducted by Zonemaster can be
+found in the [Test Requirements] document.
 
 <a name="q2"></a>
 #### 2. Who is behind Zonemaster?
-Zonemaster is a joint project between [Afnic](https://www.afnic.fr/en/)
-(registry operator of .fr TLD and several other
-TLDs, e.g. .re, .pm, .tf, .wf, .yt and .paris) and
-[The Swedish Internet Foundation](https://internetstiftelsen.se/en/)
-(registry
-operator of .se and .nu TLDs).
+Zonemaster is a joint project between [AFNIC] (registry of '.fr' TLD and several other
+TLDs, e.g. '.re', '.pm', '.tf', '.wf', '.yt' and '.paris') and the [Swedish Internet Foundation]
+(registry of '.se' and '.nu' TLDs).
 
 <a name="q3"></a>
 #### 3. How can Zonemaster help me?
 The Zonemaster tool is oriented towards two user categories:
 
   - Users who are knowledgable about the DNS protocol.
-  - Users who just want to know whether the the domains owned or used by them
+  - Users who just want to know whether the domain name they own or use
     have any issues or not.
 
 Users of the second category should contact their DNS operator
-as soon as they get the results other than "green" for any
-test of their domain name.
+in case there are errors or warnings for any test of their domain name.
 
 <a name="q4"></a>
-#### 4. Zonemaster returns "Error" or "Warning" for my domain. What does it mean?
-It depends on which test failed for your domain.
+#### 4. Zonemaster returns "Error" or "Warning" for my domain name. What does it mean?
+It depends on which test failed for your domain name.
+Each test are accompanied with one or several messages describing the issues found.
+You can also get further insight about each test from the [Implemented Test Cases] document.
```

Comment:
It is more helpful to refer to this is a link to all test cases with the test case description, `[Defined Test Cases]`.

Reply:
Updated